### DISCUSSION TITLE
How to stop it Codekit from taking control with every keystroke?

### DISCUSSION THREAD

**Discussion by SteveGuluk** [bdkjones/CodeKit#704] [General]
I used to have no problems with CodeKit but recently, I have issues with it taking control with every single key stroke ....

I primarily use CK to compile scss while working in PHPStorm.
I select the scss file and before, it would only compile on SAVE. Now, it's taking control because it sees the file has "changed" even prior to "save". It's making the feature unusable :-(

Was the compile on save removed? Or is there some other place I can change it back to the prior compile only on save, not every single keystroke... ?

**Comment by bdkjones** [comment]
It has nothing to do with CodeKit. Your IDE is set to auto-save files every few seconds after you stop typing and CodeKit is simply responding to those save events.

Change the auto-save setting in your IDE.

**Reply by SteveGuluk** [reply]
Thanks for the reply. Turns out it was a Helper App setting not auto-save. 
There were no implicit auto-saves other than if I closed the IDE or something similar.
I found the Helper App set to CodeKit and disabled that which solved the problem.
